addons/mail/static/src/components/follower/follower.js

```diff
@@ -61,8 +61,12 @@ class Follower extends Component {
      * @private
      * @param {MouseEvent} ev
      */
-    _onClickRemove(ev) {
-        this.follower.remove();
+    async _onClickRemove(ev) {
+        ev.preventDefault();
+        ev.stopPropagation();
```

Comment:
I don't understand why we need to add those.

Reply:
As mentioned above...